### PR TITLE
Migration: Popover to Functional + Hooks

### PR DESCRIPTION
**Pull request by sanyamdogra**
Similar to #618 

#### PR Checklist

- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:

Improve documentation source code visibility

#### Changes proposed in this pull request:

- Popover migrated from Class to Functional + Hooks

### CODE REVIEW

File: stories/Popover/PopoverPositionExample.js [hunk 1]
```diff
@@ -1,78 +1,69 @@
...
-          ))}
-        </span>
+const PopoverPositionExample = () => {
+  return (
+    <div className="d-flex flex-column">
+      <div className="mx-auto align-self-center p-2">
+        <PopoverItem
+          item={{ placement: 'left', text: 'a sinistra' }}
+          id={'left'}
+        />
```
Comment (dej611) [inline stories/Popover/PopoverPositionExample.js:48]: This is hidden in the source code block:

Screenshot 2020-10-28 at 09 05 42

Can you explicitly declare its content in here?

**Review by dej611 — CHANGES_REQUESTED** [review]
Left some comment in place.
Can you fix the `Interactive` version, inteegrating the knobs in the component itself - and perhaps fix the bottom behaviour (remove the padding and use flex Bootstrap classes to center the button):

Screenshot 2020-10-28 at 09 05 18

File: stories/Popover/PopoverPositionExample.js (hunk 1 above)
Comment (sanyamdogra) [reply stories/Popover/PopoverPositionExample.js]: I did the 2 mentioned changes, not sure about centralizing it without padding. For demonstration purposes, I feel it's okay. Still would love to change it. @dej611

**Review by dej611 — APPROVED** [review]
I think the fix for the inteeractive is good enough for now. Will review later on.

**Pull request merged**